trajectorytools/trajectories/concatenate.py

```diff
@@ -64,14 +66,54 @@ def _concatenate_idtrackerai_dicts(traj_dicts):
     return traj_dict_cat
 
 
-def from_several_idtracker_files(trajectories_paths, **kwargs):
+def pick_trajectory_file(session_folder):
```

Comment:
Note that during the validation of idtracker.ai trajectories using the validation GUI a trajectory files with a timestamp suffix can be generated (https://github.com/video-annotator/pythonvideoannotator-module-idtrackerai/blob/24866ee28a514a8f1b0a3186800e84280c1d4cd4/pythonvideoannotator_module_idtrackerai/models/video/objects/idtrackerai_object_io.py#L56). 
These files won't be considered by this function. 
I believe that to make it more general we should check if there are trajectory files with timestamps and take the latest one.